Tests/MapboxNavigationTests/RouteControllerSnapshotTests.swift

```diff
@@ -1,18 +1,15 @@
 import XCTest
-import FBSnapshotTestCase
 import Turf
 import MapboxDirections
 import TestHelper
 @testable import MapboxCoreNavigation
 @testable import MapboxNavigation
 
-class RouteControllerSnapshotTests: FBSnapshotTestCase {
+class RouteControllerSnapshotTests: XCTestCase {
```

Comment:
These are no longer snapshot tests, so let’s rename the class and file to RouteControllerTests.

Reply:
Thanks for the suggestion, I'll rename the file.